src/views/v2/month/tooltip/cta.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * View: Month View - Single Event Tooltip CTA
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/event/tooltip/cta.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+?>
+<div class="tribe-events-c-small-cta">
+	<a class="tribe-events-c-small-cta-link">Buy Now</a><span class="tribe-events-c-small-cta-price">$10</span>
```

Comment:
let's use classes `.tribe-events-c-small-cta__link` and `.tribe-events-c-small-cta__price` here.